openshift/templates/bcreg-x-agent/bcreg-x-agent-deploy.overrides.sh

```diff
@@ -0,0 +1,22 @@
+_includeFile=$(type -p overrides.inc)
+if [ ! -z ${_includeFile} ]; then
+  . ${_includeFile}
+else
+  _red='\033[0;31m'; _yellow='\033[1;33m'; _nc='\033[0m'; echo -e \\n"${_red}overrides.inc could not be found on the path.${_nc}\n${_yellow}Please ensure the openshift-developer-tools are installed on and registered on your path.${_nc}\n${_yellow}https://github.com/BCDevOps/openshift-developer-tools${_nc}"; exit 1;
+fi
+
+# ================================================================================================================
+# Special deployment parameters needed for injecting a user supplied settings into the deployment configuration
+# ----------------------------------------------------------------------------------------------------------------
+# The results need to be encoded as OpenShift template parameters for use with oc process.
+# ================================================================================================================
+
+# Ask the user to supply the sensitive parameters ...
+readParameter "WALLET_ENCRYPTION_KEY - Please provide the wallet encryption key for the environment.  If left blank, a 48 character long base64 encoded value will be randomly generated using openssl:" WALLET_ENCRYPTION_KEY $(generateKey) "true"
+
+_walletPrefix="BR"
+readParameter "INDY_WALLET_SEED - Please provide the indy wallet seed for the environment.  If left blank, a seed will be randomly generated using openssl:" INDY_WALLET_SEED $(generateSeed ${_walletPrefix}) "true"
+readParameter "INDY_WALLET_DID - Please provide the indy wallet did for the environment.  The default is an empty string:" INDY_WALLET_DID "" "true"
```

Comment:
Should we account for create vs update operations, like we do [here](https://github.com/bcgov/orgbook-configurations/blob/master/openshift/templates/api/api-deploy.overrides.sh)?

Reply:
@esune, I made the requested change.